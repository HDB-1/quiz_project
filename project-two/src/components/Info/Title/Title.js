import React from 'react'

export default function Title(props) {
    return (
        <div>
            <h3 id='title'>{props.title}</h3>
        </div>
    )
}
