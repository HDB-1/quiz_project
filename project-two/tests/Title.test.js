import React from 'react';
import { shallow , mount } from 'enzyme';
import Title from '../src/components/Info/Title/Title'



describe('Shallow Title', () => {
  let wrapper;
  
  beforeEach(() => wrapper = shallow(<Title title={"General Knowledge"}/>)); // this automatically tests for rendering without crashing.
  it("Should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("Should render one title tag", () => {
      expect(wrapper.find('#title').length).toEqual(1);
  });
  it("Should render the correct Title", () => {
    expect(wrapper.find('#title').text()).toEqual("General Knowledge");
  });
});
