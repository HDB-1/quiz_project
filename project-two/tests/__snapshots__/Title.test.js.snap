// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Title Should render correctly 1`] = `
<div>
  <h3
    id="title"
  >
    General Knowledge
  </h3>
</div>
`;
